PhantomBuilder/matRad_CubicVOI.m

```diff
@@ -0,0 +1,56 @@
+classdef matRad_CubicVOI < matRad_VOIVolume
+    % matRad_CubicVOI implements a class that helps to create cubic VOIs
+    %
+    % References 
+    %     -
+    %
+    % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    %
+    % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    %
+    % Copyright 2022 the matRad development team.
+    %
+    % This file is part of the matRad project. It is subject to the license
+    % terms in the LICENSE file found in the top-level directory of this
+    % distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part
+    % of the matRad project, including this file, may be copied, modified,
+    % propagated, or distributed except according to the terms contained in the
+    % LICENSE file.
+    %
+    % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    properties %additional property of cubic objects 
+        dimensions;
+    end
+    methods
+        function obj = matRad_CubicVOI(name,type,dimensions,varargin)
```

Comment:
Is there a reason dimensions are manually input? Could we use the ct as a parameter instead?

Reply:
ah sorry, misunderstood the dimensions parameter

Reply:
but if dimensions is a three-value vector, this is not a cube - it is a box then, right?

Reply:
I used a similar naming convention as in the example1 file. But box should make more sense